### DISCUSSION TITLE
Generic grid Drop-down

### DISCUSSION THREAD

**Discussion by TejaSirivuri** [generic-ui/generic-ui#21] [General]
I was working on angular and am having a grid to display but am not able to display a drop-down in the grid. Please help me in displaying a drop-downs in generic ui-grid.

**Comment by generic-ui** [comment]
Please take a look at then example that we have prepared https://generic-ui.com/examples/agile-users.

To better understand this feature read the documentation page https://generic-ui.com/guide/column-template
